tidb-cloud/data-service-manage-github-connection.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Deploy Data App Automatically with GitHub
+summary: Learn how to deploy your Data App automatically with GitHub.
+---
+
+# Deploy Data App Automatically with GitHub
+
+TiDB Cloud provides a Configuration as Code (CaC) approach to represent your whole Data App configuration as code using the JSON syntax.
+
+By connecting your Data App to GitHub, TiDB Cloud can use the CaC approach and push your Data App configurations as [configuration files](/tidb-cloud/data-service-app-config-files.md) to your preferred GitHub repository and branch.
+
+If automatic deployment is enabled for your GitHub connection, you can also modify your Data App by updating its configuration files on GitHub. After you commit the configuration file changes to GitHub, the new configurations will be deployed in TiDB Cloud automatically.
+
+In this document, you will learn how to deploy your Data App automatically with GitHub and how to manage the GitHub connection.
+
+## Before you begin
+
+Before you connect a Data App to GitHub, make sure that you have the following:
+
+- A GitHub account.
+- A GitHub repository with your target branch.
+
+> **Note:**
+>
+> The GitHub repository is used to store [Data App configuration files](/tidb-cloud/data-service-app-config-files.md) after you connect a Data App to it. If the information (such as cluster ID and endpoint URL) in the configuration files is sensitive, make sure to use a private repository instead of a public one.
```

Comment:
```suggestion
> The GitHub repository is used to store [Data App configuration files](/tidb-cloud/data-service-app-config-files.md) after you connect a Data App to it. If the information (such as the cluster ID and endpoint URL) in the configuration files is sensitive, make sure to use a private repository instead of a public one.
```